develop-organize: add overview section (#40)
open-education-hub/methodology: chapters/develop-organize/overview/reading/README.md

## Developing and Organizing Content

> Developing educational content is a time consuming activity that most educators would rather avoid.

Reviewer: ```suggestion
Developing educational content is an effort consuming activity.
It often requires educators surveying existing content, selecting and adapting topics of interest, polishing them to their needs.
```

Typically, formal education institutions from similar fields have similar curriculums.
As such, it would be much more time effective to share the educational materials used for one course rather than build them from scratch.

> To facilitate this, we propose a methodology for developing and organizing the educational materials such that the content is easily distributable, customizable, contributable and publishable.

Reviewer: ```suggestion
To facilitate this, we propose this methodology (dubbed OpenEdu Methodology) for developing and organizing the educational materials such that the content is easily to distribute, customize, contribute to and publish.
```

> Ideally, once content has been created for a particular course or field of study, it will be usable by both institutions to teach the course or by self-taught individuals for self-study.

Reviewer: ```suggestion
Ideally, once content has been created for a particular course or field of study, it will be usable both by institutions to teach the course or by self-taught individuals for self-study.
```

In this chapter, we will present the main types of content that may be created to be used in educational activities.
We discuss the strenghs and weakness for each of them and present guidelines and best practices which can be used for their creation.
Finally, we present what are the best technological tools to create, present and maintain such resources.

The topics presented in this chapter are:

- [Reading](../../reading/reading/README.md)
- [Media](../../media/reading/README.md)
- [Slides](../../slides/reading/README.md)
- [Guides](../../guides/reading/README.md)
- [Drills](../../drills/reading/README.md)
- [Projects](../../projects/reading/README.md)

> For the educational content to be available, we use a version control system (namely, `Git`) which makes it easy to store, use and propose changes to existing content.

Reviewer: ```suggestion
In order for educational content to be available, we use a version control system (namely [Git](https://git-scm.com/)), which makes it easy to store, use and propose changes to existing content.
```

> For the educational content to be available, we use a version control system (namely, `Git`) which makes it easy to store, use and propose changes to existing content.

Reviewer: I would start with some general metrics / goals / properties of developed content and the methods for developing it:

- Being able to have a history of changes.
- Publicly available
- Ability to customize
- Ability to contribute
- Automate as many actions as possible
- Easy to write and update

You mentioned them above. And for each I would then detail the use of Markdown, Git, GitHub, linters, open source tools, open content tools (reveal.md  / draw.io), open source infrastructure. And reference the infrastructure for those. That's when they would be detailed, but it's good to tie different chapters well together.

Version control systems work with repositories where all of the resources regarding a project are stored.
Therefore, a course is going to be represented by a repository.
The repository needs to respect a specific directory structure to adhere to the `Open Education Hub` methodology.
For that, it is necessary that the course is split into chapters, each chapter containing a series of topics.
Each topic stores all (or a part, depending on the field of study) of the content types that refer to that specific topic.

The general organization of a repository is the following:

```bash
|-- chapter1
|   |-- topic1
|   |-- topic2
|   |   |-- reading
|   |   |-- media
|   |   |-- slides
|   |   |-- guides
|   |   |-- drills
|   |   |-- projects
|-- chapter2
|-- chapter3
   ...

> |-- chapterN

Reviewer: ```suggestion
`-- chapterN
```
